packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarCell.tsx

```diff
@@ -14,13 +69,113 @@ export const DatepickerCalendarCell = compose<
   'button',
   DatepickerCalendarCellProps,
   DatepickerCalendarCellStylesProps,
-  ButtonProps,
+  {},
   {}
->(Button, {
-  className: datepickerCalendarCellClassName,
-  displayName: 'DatepickerCalendarCell',
-  mapPropsToStylesProps: () => ({
-    text: true,
-    size: 'small',
-  }),
-});
+>(
+  (props, ref, composeOptions) => {
+    const context = useFluentContext();
+    const { setStart, setEnd } = useTelemetry(composeOptions.displayName, context.telemetry);
+    setStart();
+
+    const { className, design, styles, variables, disabled, selected, content } = props;
+    const unhandledProps = useUnhandledProps(composeOptions.handledProps, props);
+    const ElementType = getElementType(props);
+    const getA11yProps = useAccessibility(props.accessibility, {
+      debugName: composeOptions.displayName,
+      actionHandlers: {
+        performClick: e => {
+          // prevent Spacebar from scrolling
+          e.preventDefault();
+          handleClick(e);
+        },
+      },
+      mapPropsToBehavior: () => ({
+        selected,
+        disabled,
+      }),
+      rtl: context.rtl,
+    });
+
+    const { classes } = useStyles<DatepickerCalendarCellStylesProps>(DatepickerCalendarCell.displayName, {
+      className: composeOptions.className,
+      mapPropsToStyles: () => ({
+        disabled,
+        selected,
+      }),
+      mapPropsToInlineStyles: () => ({
+        className,
+        design,
+        styles,
+        variables,
+      }),
+      rtl: context.rtl,
+      composeOptions,
+      unstable_props: props,
+    });
+
+    const handleFocus = (e: React.SyntheticEvent) => {
+      _.invoke(props, 'onFocus', e, props);
+    };
```

Comment:
```suggestion
    const handleFocus = (e: React.SyntheticEvent) => {
      _.invoke(props, 'onFocus', e, props);
    };
```

Let's remove this one until we need additional handling